test/integration/index-test.js

```diff
@@ -24,9 +24,12 @@ const {
   defaultTo
 } = require('../../src/constants');
 const { EOL } = require('os');
+const installAndGenerateBlueprint = require('../../src/install-and-generate-blueprint');
+const cliUpdateCommandModule = require('../../src/index');
+const getStartAndEndModule = require('../../src/get-start-and-end-commands');
 
 describe(function() {
-  this.timeout(30 * 1000);
+  this.timeout(30000 * 1000);
```

Comment:
what happened with this timeout? This is a super long time.

Reply:
A remnant from local testing, reverted